src/VisualStudio/Core/Def/Implementation/AnalyzerDependency/AnalyzerDependencyCheckingService.cs

```diff
@@ -80,48 +80,57 @@ internal sealed class AnalyzerDependencyCheckingService
 
             var conflicts = results.Conflicts;
             var missingDependencies = results.MissingDependencies;
+            var projects = _workspace.DeferredState.ProjectTracker.ImmutableProjects;
```

Comment:
@mavasani will it be too much trouble if it uses http://source.roslyn.io/#Microsoft.CodeAnalysis.Workspaces/Utilities/SimpleTaskQueue.cs,12 to maintain task ordering?